docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Routing with @reach/router in Gatsby v2
```

Comment:
I'd change this to focus on the subject on the article e.g. "How we improved Gatsby's accessibility in v2 by switching to @reach/router"